packages/commonwealth/client/scripts/views/pages/Editor/DragIndicator.scss

```diff
@@ -0,0 +1,9 @@
+.DragIndicator {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 10;
+  background-color: rgba(128, 128, 128, 0.5);
```

Comment:
lets use color from palette as @masvelio suggested

Reply:
Fair... I missed this one!